src/main/java/graphql/schema/GraphQLFieldDefinition.java

```diff
@@ -184,13 +162,6 @@ public Builder deprecate(String deprecationReason) {
         }
 
         public GraphQLFieldDefinition build() {
```

Comment:
@dminkovsky Ah! Yes I forgot about that.

Perhaps we define (inner or static) a no op data fetcher whose get method will throw a unsupportedoperationexception. The field definition defaults to this exception throwing no op data fetcher. When a user uses the interface they will define an updated field definition I presume and in the process re-write the data fetcher as well.

We could make it a private inner class of the field definition builder class so that no one else may use it as a data fetcher.